gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/FsBenchmark.java

```diff
@@ -185,7 +234,10 @@ private void benchmarkRead(
     CountDownLatch startLatch = new CountDownLatch(1);
     CountDownLatch stopLatch = new CountDownLatch(numThreads);
     List<Future<?>> futures = new ArrayList<>(numThreads);
+    String random_id = UUID.randomUUID().toString().substring(0, 6);
```

Comment:
can we use a more descriptive var name ?

Reply:
done